content/docs/user-guide/interactivity/robotics/_index.md

```diff
@@ -7,18 +7,19 @@ description: Learn how to create robotics simulation in Open 3D Engine (O3DE) wi
-| [Troubleshooting the simulation](troubleshooting.md)                | Helpful solutions to some of the most common issues with ROS 2.                               |
+| Topic                                                               | Description                                                                                                                       |
+| ------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| [Overview](overview.md)                                             | Overview of Gems, Templates and demos for robotics in O3DE.                                                                       |
+| [ROS 2 Project Configuration](project-configuration.md)             | Learn how to configure, build and run a project with the ROS 2 Gem.                                                               |
+| [ROS 2 Concepts and Structure](concepts-and-components-overview.md) | Overview of concepts and structure of the ROS 2 Gem, including an overview of its components.                                     |
+| [ROS 2 Simulation Interfaces](simulation-interfaces.md)             | Implementation details of ROS 2 [simulation_interfaces](https://github.com/ros-simulation/simulation_interfaces) package in O3DE. |
+| [Creating robotic simulation](creating-robotic-simulation.md)       | High-level steps to create a robotic simulation with O3DE.                                                                        |
+| [Importing robots](importing-robot.md)                              | Learn how to import robots using Robot Importer.                                                                                  |
+| [Grippers](grippers.md)                                             | Documentation of robotic grippers feature.                                                                                        |
+| [Joints Manipulation](joints-manipulation.md)                       | Joints manipulation, such as with robotic manipulator arms.                                                                       |
+| [Deploying simulation](deploying-simulation.md)                     | Building standalone simulation.                                                                                                   |
+| [Georeference](georeference.md)                                     | Link your scene with the existing location.                                                                                       |
+| [Troubleshooting the simulation](troubleshooting.md)                | Helpful solutions to some of the most common issues with ROS 2.                                                                   |
```

Comment:
This set of changes will conflict with the ones Jan made in #2651. Can you incorporate his changes and make yours on top of his, please?